### PR TITLE
[WIP] Make migrations great again.

### PR DESCRIPTION
**Pull request by cwisecarver**

### CODE REVIEW

File: addons/dropbox/models.py [hunk 1]
```diff
@@ -23,16 +23,31 @@
...
 class DropboxFile(DropboxFileNode, File):
+    # TODO DELETE ME POST MIGRATION
+    modm_model_path = 'website.files.models.dropbox.DropboxFile'
+    modm_query = None
+    # /TODO DELETE ME POST MIGRATION
     pass
 
 class Provider(ExternalProvider):
-
+    # TODO DELETE ME POST MIGRATION
```
Comment (mfraezz) [inline addons/dropbox/models.py]: Also not stored

File: addons/box/models.py [hunk 2]
```diff
@@ -23,15 +23,31 @@
...
 
 class BoxFile(BoxFileNode, File):
+    # TODO DELETE ME POST MIGRATION
+    modm_model_path = 'website.files.models.box.BoxFile'
+    modm_query = None
+    # /TODO DELETE ME POST MIGRATION
     pass
 
 class Provider(ExternalProvider):
+    # TODO DELETE ME POST MIGRATION
```
Comment (mfraezz) [inline addons/box/models.py]: Also not stored

File: addons/github/models.py [hunk 3]
```diff
@@ -45,6 +53,10 @@ def touch(self, auth_header, revision=None, ref=None, branch=None, **kwargs):
 
 
 class GitHubProvider(ExternalProvider):
+    # TODO DELETE ME POST MIGRATION
```
Comment (mfraezz) [inline addons/github/models.py]: Also not stored

File: addons/googledrive/models.py [hunk 4]
```diff
@@ -27,17 +28,33 @@
...
 class GoogleDriveFile(GoogleDriveFileNode, File):
+    # TODO DELETE ME POST MIGRATION
+    modm_model_path = 'website.files.models.googledrive.GoogleDriveFile'
+    modm_query = None
+    # /TODO DELETE ME POST MIGRATION
     pass
 
 
 class GoogleDriveProvider(ExternalProvider):
+    # TODO DELETE ME POST MIGRATION
```
Comment (mfraezz) [inline addons/googledrive/models.py]: Also not stored

File: addons/mendeley/models.py [hunk 5]
```diff
@@ -2,22 +2,25 @@
...
 from website.addons.mendeley.api import APISession
+from website.addons.mendeley.serializer import MendeleySerializer
 from website.citations.providers import CitationsOauthProvider
 from website.util import web_url_for
 
-from framework.exceptions import HTTPError
-
 
 class Mendeley(CitationsOauthProvider):
+    # TODO DELETE ME POST MIGRATION
```
Comment (mfraezz) [inline addons/mendeley/models.py]: Also not stored

File: addons/owncloud/models.py [hunk 6]
```diff
@@ -1,35 +1,47 @@
...
 
 class OwncloudFile(OwncloudFileNode, File):
+    # TODO DELETE ME POST MIGRATION
+    modm_model_path = 'website.files.models.owncloud.OwncloudFile'
+    modm_query = None
+    # /TODO DELETE ME POST MIGRATION
     pass
 
 class OwnCloudProvider(BasicAuthProviderMixin):
+    # TODO DELETE ME POST MIGRATION
```
Comment (mfraezz) [inline addons/owncloud/models.py]: Also not stored

File: addons/zotero/models.py [hunk 7]
```diff
@@ -16,6 +15,10 @@
 MAX_CITATION_LOAD = 200
 
 class Zotero(CitationsOauthProvider):
+    # TODO DELETE ME POST MIGRATION
```
Comment (mfraezz) [inline addons/zotero/models.py]: Also not stored

**Comment by cwisecarver** [comment]
Merging so that travis has many many test matrices.

**Comment by icereval** [comment]
👍

**Pull request merged**